pkg/appfile/storage/storage.go

```diff
@@ -1,37 +1,44 @@
 package storage
 
 import (
+	"errors"
+	"os"
+
 	"github.com/oam-dev/kubevela/pkg/appfile/storage/driver"
 )
 
 // Store application store client
 var Store *Storage
 
-func init() {
-	// TODO support provide multiple ways:
-	// system environment
-	// system configfile
-	// startup arguments
-	Store = NewStorage(driver.LocalDriverName)
-}
-
 // Storage is common storage client，use it to get app and others resource
 type Storage struct {
 	driver.Driver
 }
 
+func init() {
+	// default use Local Storage driver
+	_ = NewStorage(driver.LocalDriverName)
+}
+
 // NewStorage form driver type
-func NewStorage(driverName string) *Storage {
-	// TODO remove driverName param ,should use environment get it
-	// FIXME use env to get user storageDriver
+// 1. get system environment from STORAGEDRIVER
+// 2. startup arguments from storage-driver
+func NewStorage(driverName string) error {
```

Comment:
Why not return the Storage just like the original function? Assigning a global variable in the function can lead to overriding the variable unintentionally if the NewStorage function is called multiple times.

Reply:
We need to support the storage mode specified by the `--storage-driver` flag, so we can't get parameters directly in init.

It would be nice if `NewStorage ` could be made private.